jenkins-integration/beaker/install/pe/98_sync_codedir.rb

```diff
@@ -1,27 +0,0 @@
-test_name 'Sync code-staging to code via File Sync'
```

Comment:
I mentioned this in #155 as well, but this is used in the PE automated gatling tests still and should not be removed.

Reply:
@haus thanks!  can you give us some context on when/how those are run?
we need to start adding tests around these so we don't break you.

Reply:
it is called in `jenkins-integration/jenkins-jobs/common/scripts/job-steps/055_perform_file_sync.sh`, which is in turn part of `jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy` which is shared by all of the jenkins job. The PE job i mention is a weekly run against PE. We have currently disabled it until we have something ready to validate against irving, but this is an example of the job in question: http://pu5a2h8b9gtw39k.delivery.puppetlabs.net:8080/job/pe-puppetserver-latest/
The pipeline checks to see if the version supports file sync and if it is enabled, and if both are true a sync is performed as part of the setup.

Reply:
@er0ck So far, all of the changes from silver have been in the setup and config directories, which won't affect us at all. Changes/removals from the other directories (gatling-raw, jenkins-integration, proxy-recorder, and simulation-runner) have the potential to affect us and we should probably give any PR touching those directories a review.

Reply:
Yeah, not even sure how that got deleted, that happened somehow when I merged.  This should only be touching the install_gatling directory, not anything under jenkins-integration.